pom.xml

```diff
@@ -244,6 +243,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>3.2.0</version>
+                <executions>
+                    <execution>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>src/main/kotlin</source>
```

Comment:
Do we need `src/main/java` here as well? What's grabbing those sources?

Reply:
No, the Java sources are taken by default. maven-sources-plugin consumes what is known in the Maven Reactor as sources, and unfortunately the kotlin-maven-plugin doesn't mark src/main/kotlin as sources.

Reply:
Ok, and this setting won't override the existing one, right?  I.e. now only do the kotlin sources?

Reply:
No, of course not.